README.md

```diff
@@ -86,6 +86,10 @@ Arbiter treats Rust smart-contract bindings as first-class citizens. The contrac
 `arbiter bind` wraps `forge` with some convenience features that will generate all your bindings to src/biindings as a rust module. 
 [Foundry](https://github.com/foundry-rs/foundry) power-users are welcome to use `forge` directly.
 
+### Optional Arguments
+
+You can run `arbiter init -o` followed by any optional arguments. Currently the only optional argument is `no-git` that will be sure to remove the `.git` directory from the template upon initialization.
```

Comment:
Can we make it so you can just run `arbiter init -no-git` as opposed to `arbiter init -o -no-git`?

Reply:
Yes! Just pushed these changes now